baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/MoreASTHelpers.java

```diff
@@ -123,5 +129,39 @@ public static Type getResultType(Tree tree) {
                 : ASTHelpers.getType(tree);
     }
 
+    public static boolean isSealed(@Nullable ClassSymbol classSymbol) {
+        if (classSymbol == null) {
+            return false;
+        }
+        long flags = classSymbol.flags();
+        return (flags & (1L << 62)) != 0;
```

Comment:
are there JDK docs for this?

Reply:
Yes, in `com.sun.tools.javac.code.Flags`, but only in newer releases, where we target jdk-11